docs/en/observability/whats-new.asciidoc

```diff
@@ -3,19 +3,248 @@
+
+For {minor-version}, we are delighted to announce that {es} now supports creating
+runtime fields in the {stack}. Once a runtime field is created, for example, the
+`day_of_week` field, you can view and filter them in the {logs-app}'s stream page,
+just like standard fields.
+
+For more information, see our latest https://elastic.co/blog/introducing-elasticsearch-runtime-fields[blog].
+
+[role="screenshot"]
+image::images/runtime-fields.png[Runtime fields in logs]
+
+[discrete]
+== {fleet} and {agent} (Beta 3)
+
+In {minor-version}, we’re excited to release our third beta of {fleet} and {agent}.
+So you can identify and troubleshoot issues faster, we have added better observability
+to {agent} integrations, logs, and metrics. This helps you gain visibility into
+issues with {agent} right inside the {fleet} app.
+
+The agent details page now shows information about the agent settings, the host, and
+which integrations are running.
+
+[role="screenshot"]
+image::images/elastic-agent.png[Elastic agent page]
+
+Additionally, the status information from the {agent} is now available in {es}.
+We added a simple log viewer that lets you easily search and filter. It is only compatible
+with {agent}s version {minor-version} or higher. For a richer experience, you can also open
+the {logs-app} to view the logs.
+
+[role="screenshot"]
+image::images/elastic-agent-logs.png[Elastic agent logs]
+
+For more in-depth insight when troubleshooting, we’ve made it easier to debug logging.
+At the bottom of the page, you can change the log level for the agent. You can
+apply it selectively on a per-agent basis to avoid sending a high volume of data from all your agents.
+
+We’ve also improved agent binary upgrades. To ensure that the agent remains functioning
+and reachable from the {fleet} app, we will automatically roll the agent back to the
+previous version during an upgrade, if it has an error.
+
+Finally, {agent} can self-protect when the {endpoint-sec} integration is enabled.
+Self-protection means that {endpoint-sec} guards against users and attackers interfering
+with {agent}’s functionality. Over time, we will enhance these guards to prevent
+interference with {agent}.
+
+[discrete]
+== APM Prometheus metrics client
+
+In {minor-version}, the APM Python agent learned a new trick! The latest version can
+automatically detect and monitor Prometheus metrics by auto-instrumentation of the
+Prometheus Python library.
+
+Your application custom metrics are now monitored with zero
+effort whenever you monitor your Python applications with APM. Once you hook up custom
+metrics, you can quickly build a {kib} dashboard using TSVB or Lens to analyze them
+and correlate with other performance metrics.
+
+[discrete]
+== {log-driver-long} (GA)
+
+In {minor-version}, we are excited to announce that the https://hub.docker.com/plugins/elastic-logging-plugin[Docker logging plugin]
+for enabling simpler UX for application logging is now generally available.
```

Comment:
will all readers know what we mean by UX? Might be better to write it out just in case.